src/index.tsx

```diff
@@ -867,10 +871,12 @@ export const Content = React.forwardRef<HTMLDivElement, ContentProps>(function (
     }
   }, []);
 
-  function handleOnPointerUp(event: React.PointerEvent<HTMLDivElement>) {
+  function handleOnPointerUp(event: React.PointerEvent<HTMLDivElement> | null) {
     pointerStartRef.current = null;
     wasBeyondThePointRef.current = false;
-    onRelease(event);
+    if (event) {
```

Comment:
this is the actual fix to the opened issue

Reply:
I'm not 100% sure on this one, the safest approach would be to handle the null check inside of the `onRelease` so we can still set isDragging to false, lmk what you prefer.

Reply:
Let's go with the safer approach.

Reply:
@emilkowalski done, but the playwright is failing even do it shows "success", I'm investigating the regression: https://github.com/emilkowalski/vaul/actions/runs/11240999977/job/31251582106?pr=477

Reply:
looks like intermittent, locally it passes (not always)... maybe related to the animation timeout?
![image](https://github.com/user-attachments/assets/43fe40e1-6379-412d-ad8a-23646dd37f46)

Reply:
adding `await page.waitForTimeout(ANIMATION_DURATION);` to the first line of `openDrawer()` made it consistent locally, but I won't commit it.

Reply:
only flask/intermittent, nothing that I could find, lmk if you want me to check a specific case.